.github/workflows/renovate.yml

```
@@ -0,0 +1,22 @@
+      - 'renovate.json'
+  pull_request:
+    paths:
+      - 'renovate.json'
+
+jobs:
+  renovate-config-validator:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository)
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)